src/main/resources/META-INF/resources/webjars/form-flow/0.0.1/css/libraryStyles.css

```diff
@@ -393,6 +393,10 @@ body,
   color: #FFFFFF;
 }
 
+.button--other {
```

Comment:
used by `Document Recommendations` page.  Not sure if this is a good name or not, but needed to update the margin to get it to look correct.